e2e/hardhatTransactionFlowSendAndWC.spec.js

```diff
@@ -47,31 +49,10 @@ const getOnchainBalance = async (address, tokenContractAddress) => {
 };
 
 beforeAll(async () => {
-  // Connect to hardhat
-  await exec('yarn hardhat');
-  await exec(
-    'open /Applications/Xcode.app/Contents/Developer/Applications/Simulator.app/'
-  );
+  await Helpers.startHardhat();
+  await Helpers.startIosSimulator();
```

Comment:
I don't like this naming. It sounds like it's gonna start iOS simulator even tho we're on andorid.